docs/examples/wordcount/server_test.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/suborbital/vektor/vk"
+	"github.com/suborbital/vektor/vtest"
+)
+
+// Lets us reuse the same *vk.Server instance for multiple tests and requests
+var vt *vtest.VTest
+
+func init() {
+	server := vk.New(vk.UseAppName("wordcount_testing"))
+	attachRoutes(server)
```

Comment:
Instead of creating a server and then calling `attachRoutes`, I'd suggest having a single `setupServer` function which is called in the example and in the tests, which shows the user that they're truly testing their exact production code, not a special test server.